ArcGISRuntimeSDKQt_QMLSamples/Analysis/FindServiceAreasForMultipleFacilities/FindServiceAreasForMultipleFacilities.qml

```diff
@@ -0,0 +1,147 @@
+// [WriteFile Name=FindServiceAreasForMultipleFacilities, Category=Analysis]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.8
+import QtQuick.Layouts 1.11
+import QtQuick.Controls 2.2
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property url featureTableUrl: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0"
+    property url imageUrl: "https://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png"
+    property url serviceTaskAreaUrl: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ServiceArea"
+    property list<SimpleFillSymbol> fillSymbols: [
+        SimpleFillSymbol {
+            style: Enums.SimpleFillSymbolStyleSolid
+            color: "#66FFA500"
```

Comment:
named color

Reply:
The colors used aren't predefined colors